nh_eobs/tests/common/test_data_creator.py

```diff
@@ -5,15 +5,17 @@ def admit_and_place_patient(self):
 
 
 def admit_patient(self):
-    self.api_model = self.env['nh.eobs.api']
+    try:
+        self.api_model = self.env['nh.eobs.api']
+    except KeyError:
+        self.api_model = self.env['nh.clinical.api']
```

Comment:
I wanted to be able to use it from multiple different levels so if `nh.eobs.api` was available I figured we should use it so we don't skip logic that might affect the outcome of the test.